notebooks/week6/04.create_alert.py

```
@@ -0,0 +1,47 @@
+            query_id=query.id
+        )
+    )
+
+
+
+
+# COMMAND ----------
+
+# cleanup
+w.queries.delete(id=query.id)
+w.alerts.delete(id=alert.id)
```

Ruff (0.7.0)

No newline at end of file

Add trailing newline

(W292)